cg/store/filters/status_sample_filters.py

```diff
@@ -87,9 +87,9 @@ def filter_samples_do_not_invoice(samples: Query, **kwargs) -> Query:
     return samples.filter(Sample.no_invoice.is_(True))
 
 
-def filter_samples_by_customer_id(samples: Query, customer_id: str, **kwargs) -> Query:
+def filter_samples_by_customer_id(samples: Query, customer_ids: List[int], **kwargs) -> Query:
```

Comment:
Do you not also have to update the call to this function and the apply_function to not brake the code?